ocaml/xapi/extauth_plugin_ADpbis.ml

```diff
@@ -690,11 +753,8 @@ struct
         try
           let (_: (string*string) list) =
             pbis_common_with_password
-              pass
-              "/opt/pbis/bin/domainjoin-cli"
-              (["join"]
-               @ ou_params @ disabled_module_params @
-               ["--ignore-pam";"--notimesync";domain;user])
+              pass domain_join_cli_path
+              (["join"] @ ou_params @ disabled_module_params @ ["--ignore-pam";"--notimesync";domain;user])
```

Comment:
Use `List.concat` and avoid `@` - it can lead to subtle bugs in connection with `if..then..else`

Reply:
updated